response_operations_ui/views/respondents.py

```diff
@@ -14,22 +17,84 @@
                           static_folder='static', template_folder='templates')
 
 
-@respondent_bp.route('/', methods=['GET', 'POST'])
+@respondent_bp.route('/', methods=['GET'])
+@login_required
+def respondent_home():
+    return render_template('respondent-search/respondent-search.html',
+                           form=RespondentSearchForm(),
+                           breadcrumbs=[{"title": "Respondents"}])
+
+
+@respondent_bp.route('/search', methods=['POST'])
+@login_required
+def search_redirect():
+    form = RespondentSearchForm()
+    form_valid = form.validate()
+
+    if not form_valid:
+        flash('At least one input should be filled')
+
+    source = form.source.data or 'home'
+    page = request.args.get('page', 1)
+
+    if not form_valid:
+        return redirect(url_for('respondent_bp.respondent_home'))
+
+    return redirect(url_for('respondent_bp.respondent_search',
+                            email_address=form.email_address.data or '',
+                            first_name=form.first_name.data or '',
+                            last_name=form.last_name.data or '',
+                            source=source,
+                            page=page))
+
+
+@respondent_bp.route('/search')
+@respondent_bp.route('/search/', methods=['GET'])
```

Comment:
These 2 lines should become just:
`@respondent_bp.route('/search', methods=['GET'])`

because you get urls that look like `<host>/respondents/search/?email_address=&first_name=Person&last_name=&page=1` otherwise